README.md

```diff
@@ -25,6 +20,35 @@ View the [application examples](core/examples) to explore Luigi's features.
 
 For details, see [Luigi documentation](docs/README.md).
 
+## Browser support
+
+If you want to support Internet Explorer 11 in your application, update your Luigi imports as follows:
+### Luigi Core
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel='stylesheet' href='core/public/luigi-ie11.css'>
```

Comment:
For your example the app is placed in the `core` folder, so the path should be like this `<link rel='stylesheet' href='../public/luigi-ie11.css'>`. But I'm not sure if it will be a general case or not :)

Reply:
Good point, I will change the paths to look as in our angular app, which is consuming the npm package directly